package.json

```diff
@@ -3,13 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "tsc",
+    "build": "npm run lint && tsc",
     "start": "npm run lint && run-p lint:watch build:watch",
     "test": "npm run test:all",
     "test:integration": "jest --config jest/integration.config.js",
     "test:unit": "jest --config jest/unit.config.js",
     "test:all": "jest",
-    "lint": "tslint -t stylish 'src/**/*.ts' --force",
+    "lint": "tslint -t stylish 'src/**/*.ts'",
```

Comment:
calling `npm start` with lint errors now breaks the devmode due to `--force` being remove from `lint` task. this is undesired as we want to avoid _blocking_ the developer with linting issues during the development process. linting should only fail the `build` task.